### DISCUSSION TITLE
Number of thickness constraints

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#280] [General]
Hi,

How do we define the number of thickness constraints? In the papers I read, for example, it is said that number of thickness constraint is 200, but I have no idea how it is calculated does it relate to the number of FFD points?

**Comment by friedenhe** [comment]
This is covered in the workshop slides and video.

**Reply by hosseinnegahban** [reply]
I have watched the complete workshop and also checked the slides but did not find the answer. I do not mean the number of FFD points. My question is regarding the number of thickness constraints 
![InkedUntitled_LI](https://user-images.githubusercontent.com/97816713/155827447-228033cd-4ec0-4355-abbd-4358f4391702.jpg)

**Reply by friedenhe** [reply]
It is in workshop slides ~79. Also check this: https://mdolab-mach-aero.readthedocs-hosted.com/en/latest/machAeroTutorials/opt_aero.html#geometric-constraints
